### PR TITLE
Update version and CHANGELOG for new release: v0.0.3

### PR DESCRIPTION
**Pull request by xydesa**
Signed-off-by: Alexander Xydes <alexander.xydes@navy.mil>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
+
+### Changed
+
+- Filtering out yaml files from the dockerfile sources since all three dockerfile linters
+  use yaml files as their config files. (#12)
+
+### Fixed
+
+- Changing dockerfilelint rc file to an empty file since dockerfilelint doesn't handle an empty rules list.
+  Adding issues to the statick output if one of the dockerfile linter plugins throws an error while parsing
```
Comment (denewiler) [inline CHANGELOG.md]:
```suggestion
- Adding issues to the statick output if one of the dockerfile linter plugins throws an error while parsing
```

Looks like this should be a separate item in the list.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Removed
 
+## v0.0.3 - 2021-09-29
+
+### Changed
+
+- Filtering out yaml files from the dockerfile sources since all three dockerfile linters
+  use yaml files as their config files. (#12)
```
Comment (denewiler) [inline CHANGELOG.md]:
```suggestion
  use yaml files as their configuration files. (#12)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
+
+### Fixed
+
+- Changing dockerfilelint rc file to an empty file since dockerfilelint doesn't handle an empty rules list.
+  Adding issues to the statick output if one of the dockerfile linter plugins throws an error while parsing
+  the output from it's tool. (#11)
```
Comment (denewiler) [inline CHANGELOG.md]: I'm not sure if this change is correct.

```suggestion
  the output from its tool. (#11)
```

File: CHANGELOG.md (hunk 1 above)
Comment (xydesa) [reply CHANGELOG.md]: That's actually part of the same item and same ticket as the line above it. I'm ok separating it into a separate item if you still think I should.

**Review by denewiler — APPROVED** [review]

**Pull request merged**